doc/source/reference/c-api/array.rst

```diff
@@ -4340,6 +4410,40 @@ Enumerated Types
        :c:data:`NPY_STABLESORT` are aliased to each other and may refer to one
        of several stable sorting algorithms depending on the data type.
 
+.. c:enum:: NPY_SORT_NAN_POSITION
+
+   An enum used to indicate the position of NaN values in sorting.
+
+   .. c:enumerator:: NPY_SORT_NAN_TO_START
+
+      Indicates that NaN values should be sorted to the start.
+
+   .. c:enumerator:: NPY_SORT_NAN_TO_END
+
+      Indicates that NaN values should be sorted to the end.
+
+.. c:enum:: NPY_COMPARE_RESULT
+
```

Comment:
IIRC, C enums may be unsigned depending on whether they are explicitly defined. Are these actually enums, or just here for markup?

Reply:
They are actually enums, defined in `dtype_api.h`!